pkg/ovs/ovsctl/ofctl.go

```diff
@@ -63,6 +64,27 @@ func (c *ovsCtlClient) DumpTableFlows(table uint8) ([]string, error) {
 	return c.DumpFlows(fmt.Sprintf("table=%d", table))
 }
 
+func (c *ovsCtlClient) DumpGroup(groupID int) (string, error) {
+	// There seems a bug in ovs-ofctl that dump-groups always returns all
+	// the groups when using Openflow13, even the group ID is provided. As
```

Comment:
s/even the group ID is provided/even when the group ID is provided

Reply:
Done.